src/test/ProposalPayloadV2E2E.t.sol

```diff
@@ -0,0 +1,264 @@
+                stableRateSlope2: OLD_INTEREST_RATE_STRATEGY_POLYGON.stableRateSlope2(),
+                variableRateSlope1: 14 * (AaveV2Helpers.RAY / 100),
+                variableRateSlope2: 300 * (AaveV2Helpers.RAY / 100)
+            }),
+            POLYGON
+        );
+
+        createConfigurationSnapshot("post-AaveV2Polygon-interestRateUpdate", AaveV2Polygon.POOL);
+    }
+
+    // Interest Strategy Ethereum
+
+    function testUtilizationAtZeroPercentEthereum() public {
+        (uint256 liqRate, uint256 stableRate, uint256 varRate) = strategy.calculateInterestRates(
+            CRV,
+            100 * 1e18,
+            0,
+            0,
+            68200000000000000000000000,
+            2000
+        );
+
+        // At nothing borrowed, liquidity rate should be 0, variable rate should be 3% and stable rate should be 3%.
+        assertEq(liqRate, 0);
+        assertEq(stableRate, 3 * (AaveV2Helpers.RAY / 100));
+        assertEq(varRate, 3 * (AaveV2Helpers.RAY / 100));
+    }
+
+    function testUtilizationAtOneHundredPercentEthereum() public {
+        (uint256 liqRate, uint256 stableRate, uint256 varRate) = strategy.calculateInterestRates(
+            CRV,
+            0,
+            0,
+            100 * 1e18,
+            68200000000000000000000000,
+            2000
+        );
+
+        // At max borrow rate, stable rate should be 313% and variable rate should be 317%.
+        assertEq(liqRate, 2536000000000000000000000000);
+        assertEq(stableRate, 3130000000000000000000000000);
+        assertEq(varRate, 3170000000000000000000000000);
+    }
+
+    function testUtilizationAtUOptimalEthereum() public {
+        (uint256 liqRate, uint256 stableRate, uint256 varRate) = strategy.calculateInterestRates(
+            CRV,
+            30 * 1e18,
+            0,
+            70 * 1e18,
+            68200000000000000000000000,
+            2000
+        );
+
+        // At UOptimal, stable rate should be 13% and variable rate should be 17%.
+        assertEq(liqRate, 95200000000000000000000000);
+        assertEq(stableRate, 13 * (AaveV2Helpers.RAY / 100));
+        assertEq(varRate, 17 * (AaveV2Helpers.RAY / 100));
+    }
+
+    // Interest Strategy Polygon V2
+
+    function testUtilizationAtZeroPercentPolygonV2() public {
+        vm.selectFork(polygonFork);
+        (uint256 liqRate, uint256 stableRate, uint256 varRate) = strategyPolygon.calculateInterestRates(
+            CRV_POLYGON,
+            100 * 1e18,
+            0,
+            0,
+            68200000000000000000000000,
+            2000
+        );
+
+        // At nothing borrowed, liquidity rate should be 0, variable rate should be 3% and stable rate should be 0%.
+        assertEq(liqRate, 0);
+        assertEq(stableRate, 0 * (AaveV2Helpers.RAY / 100));
+        assertEq(varRate, 3 * (AaveV2Helpers.RAY / 100));
+    }
+
+    function testUtilizationAtOneHundredPercentPolygon() public {
```

Comment:
Small nitpick. If your other functions are going to be called PolygonV2, then keep the same naming here as well.